dlrover/python/master/watcher/k8s_watcher.py

```diff
@@ -90,18 +92,34 @@ def _convert_pod_event_to_node_event(event):
         # We only care about pod related events
         return None
 
+    metadata: client.V1ObjectMeta = evt_obj.metadata
+    job_name = metadata.labels[ElasticJobLabel.JOB_KEY]
+
     # Skip events of dlrover mater Pod
-    pod_type = evt_obj.metadata.labels[ElasticJobLabel.REPLICA_TYPE_KEY]
+    pod_type = metadata.labels[ElasticJobLabel.REPLICA_TYPE_KEY]
     if pod_type == NodeType.DLROVER_MASTER:
         return None
 
-    metadata: client.V1ObjectMeta = evt_obj.metadata
     rank = int(metadata.labels[ElasticJobLabel.RANK_INDEX_KEY])
     pod_id = int(metadata.labels[ElasticJobLabel.REPLICA_INDEX_KEY])
     pod_name = metadata.name
     host_name = evt_obj.spec.node_name
     host_ip = evt_obj.status.host_ip
 
+    # Skip event of pod deleted if the deleted pod already exist for
```

Comment:
`Skip deleted event of pod if the cluster has relaunched a new pod with the same type and rank as the deleted pod.`

Reply:
done